Controller/src/cz/fidentis/controller/data/TransparencyData.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fidentis.controller.data;
+
+import java.awt.Color;
+
+/**
+ *
+ * @author xferkova
+ */
+public class TransparencyData {

```

Comment:
Mozno len poznamka k pomenovaniu - mne osobne by tam viac sedelo nieco ako `TransparencyConfig` alebo podobne, aby to na prvy pohlad hovorilo ze to je objekt ktory popisuje nastavenie/konfiguraciu nejakej vizualizacie. Data mi pride moc vseobecne a nevystihujuce ten ucel. (To iste plati aj pre `VectorsData` a `CrosscutData`)